chef-config/lib/chef-config/config.rb

```diff
@@ -490,6 +490,16 @@ def self.path_accessible?(path)
     # HTTP file servers.
     default(:trusted_certs_dir) { PathHelper.join(config_dir, "trusted_certs") }
 
+    # A directory that contains additional configuration scripts to load for chef-client
+    default(:client_d_dir) { PathHelper.join(config_dir, "client.d") }
+
+    # A directory that contains additional configuration scripts to load for solo
+    default(:solo_d_dir) { PathHelper.join(config_dir, "solo.d") }
+
+    # A directory that contains additional configuration scripts to load for
+    # the workstation config
+    default(:conf_d_dir) { PathHelper.join(config_dir, "conf.d") }
```

Comment:
`knife.rb` is very softly deprecated. There's no real upside to forcing people to switch, but we decided to use `config.rb` going forward because it's used by many tools other than knife.